ivy/functional/frontends/torch/loss_functions.py

```diff
@@ -2,6 +2,42 @@
 import ivy
 
 
+def _get_reduction_method(reduction, to_reduce):
+    if reduction == "none":
+        ret = to_reduce
+    elif reduction == "mean":
+        ret = ivy.mean(to_reduce)
+    elif reduction == "elementwise_mean":
```

Comment:
This is never tested, why is it in if its not a documented parameter

Reply:
This one is deprecated in Pytorch and am going to remove it from here.